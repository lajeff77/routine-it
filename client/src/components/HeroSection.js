import React from 'react';
import { Link } from 'react-router-dom'


const HeroSection = () => {
    return(
        <div className='hero-section'>
            <h1>Everyone should be able to keep a routine!</h1>
            <Link to='/signup'><button>Get Started!</button></Link>
        </div>
    )
}

export default HeroSection;