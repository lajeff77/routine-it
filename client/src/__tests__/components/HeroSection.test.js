import { render} from "@testing-library/react";
import React from "react";
import HeroSection from "../../components/LandingPage/HeroSection";

describe(HeroSection.name, ()=>{
    test('HeroSection matches snapshot', () => {
        const { container } = render(<HeroSection/>);
        expect(container).toMatchSnapshot();});
});