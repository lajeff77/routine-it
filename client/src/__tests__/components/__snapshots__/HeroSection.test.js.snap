// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroSection HeroSection matches snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-66os1a"
  >
    <h1
      class="MuiTypography-root MuiTypography-h1 css-1q53ye7-MuiTypography-root"
    >
      Everyone should be able to keep a routine!
    </h1>
    <a
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-disableElevation MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-disableElevation css-jv9d4y-MuiButtonBase-root-MuiButton-root"
      href="/signup"
      tabindex="0"
    >
      Get Started!
      <span
        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
      />
    </a>
  </div>
</div>
`;
